Jenkinsfiles/hydra_copy

```diff
@@ -154,6 +154,23 @@ pipeline {
                             } catch (err) {
                                 println("Something went wrong at post processing of: ${buildId} failed: ${err}")
                             }
+                        } else if ("${buildData['Job']}".contains("fmo-os")) {
+                            try {
+                                build job: "${processingJob}", parameters: [
+                                    [$class: 'StringParameterValue', name: 'outputPath', value: "${outputPath}"],
+                                    [$class: 'StringParameterValue', name: 'buildID', value: "${buildId}"],
+                                    [$class: 'StringParameterValue', name: 'resultsPath', value: "${resultsPath}"],
+                                    [$class: 'StringParameterValue', name: 'server', value: "${params.server}"],
+                                    [$class: 'StringParameterValue', name: 'project', value: "${buildData['Project']}"],
+                                    [$class: 'BooleanParameterValue', name: 'development', value: params.development],
+                                    [$class: 'BooleanParameterValue', name: 'hw_tests', value: false],
+                                    [$class: 'StringParameterValue', name: 'label', value: "${params.label}"],
+                                    [$class: 'StringParameterValue', name: 'device', value: "fmo-os"],
```

Comment:
LGTM
Lot of copy code though, and as only 1 parameter seems to be changing would it be better to only change a value of that one variable inside a switch/if clause and then call another job by using the value of that variable as a value for the 'device' parameter when launching another job.